### ISSUE TITLE
bad card numbers JOU set

### ISSUE THREAD

**Issue by Artheos40** [gaddlord/mtg.studio#496]
while having a filter to get only JOU set, some cards have a * in their car numbers 
which makes em get sorted first instead of with their respective numbers.
you just have to set a filter to see it. 

added to that, doomwake giant [promo]  has a bad  card number too . the set is has 165 cards not 185. (all seen in capture)

![Capture d’écran 2023-04-20 174956](https://user-images.githubusercontent.com/119977998/233495622-fffd0716-987b-4b33-aec6-15d535a6d36a.png)

img of the filter just to show : 

![Capture d’écran 2023-04-20 175749 filter](https://user-images.githubusercontent.com/119977998/233496367-29621bde-ca99-4341-b6e2-d82e96e3ae69.png)

**Comment by Artheos40** [comment]
same goes for BNG and THS sets. i'm going backwards in the sets and i'll keep on the lookout going backwards..
here'S a pic for BNG then for THS
Born of the gods (BNG)
![Capture d’écran 2023-04-20 192643 BNG](https://user-images.githubusercontent.com/119977998/233507212-ce1eb3b0-5c1e-412f-95d1-7fe72c20573e.png)
Theros (THS)
![Capture d’écran 2023-04-20 192714 THS ](https://user-images.githubusercontent.com/119977998/233507240-07f1337d-becd-4500-9ac9-f14ccf246bbb.png)

**Comment by gaddlord** [comment]
RE: Cards with "a" in the Collector No not being sorted properly. WotC have gone long miles to make the collector no hard to parse. For this reason in version 4.x of MTG Studio we introduced a new hidden column "Sortable Collector No" which is only a number extracted from the "Collector No". This column is used for sorting the Collector No. So far we do not have plans to introduce to versions 3.x.

RE: Doomwake Giant [Promo] is fixed to have the correct Collector No "*66/165 Prerelease" in forthcoming version 3.182.
